src/components/Translate/Translate.react.js

```diff
@@ -14,22 +14,8 @@ import nl from './nl.json';
 import pb from './pb.json';
 import np from './np.json';
 
-class Translate extends Component {
-  constructor(props) {
-    super(props);
-    this.state = {
-      text: this.props.text,
-      defaultPrefLanguage: UserPreferencesStore.getPrefLang(),
-    };
-  }
-
-  changeLanguage = text => {
-    this.setState({
-      text: text,
-    });
-  };
-
-  lang = {
+const Translate = props => {
```

Comment:
Use `(props)`

Reply:
Sir, Eslint is changing is back to props as soon as i use git commit